### PR TITLE
refactor(components): [statistic]

### PR DESCRIPTION
**Pull request by holazz**
**Screenshot**

1.

preview-10939-element-plus surge sh_en-US_component_statistic html (2)

2.

preview-10939-element-plus surge sh_en-US_component_statistic html

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/countdown/src/countdown.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    let diff = timestamp - Date.now()
+    emit('change', diff)
+    if (diff <= 0) {
+      diff = 0
+      stopTimer()
+      emit('finish')
+    }
+    rawValue.value = diff
+  }, REFRESH_INTERVAL)
+}
```
Comment (kooriookami) [inline packages/components/countdown/src/countdown.vue:57]:
```suggestion
import { computed, onBeforeUnmount, ref, watch } from 'vue'
import { ElStatistic } from '@element-plus/components/statistic'
import { cAF, rAF } from '@element-plus/utils/raf'
import { countdownEmits, countdownProps } from './countdown'
import { formatTime, getTime } from './utils'

defineOptions({
  name: 'ElCountdown',
})

const props = defineProps(countdownProps)
const emit = defineEmits(countdownEmits)

let timer: ReturnType<typeof setInterval> | undefined

const rawValue = ref(getTime(props.value) - Date.now())

const displayValue = computed(() => formatTime(rawValue.value, props.format))

const formatter = (val: number) => formatTime(val, props.format)

const stopTimer = () => {
  if (timer) {
    cAF(timer)
    timer = undefined
  }
}

const startTimer = () => {
  const timestamp = getTime(props.value)

  const frameFunc = () => {
    let diff = timestamp - Date.now()
    emit('change', diff)
    if (diff <= 0) {
      diff = 0
      stopTimer()
      emit('finish')
    } else {
      timer = rAF(frameFunc)
    }
    rawValue.value = diff
  }
  timer = rAF(frameFunc)
}
```

I think use rAF is better.

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: packages/components/countdown/src/countdown.vue (hunk 1 above)
Comment (holazz) [reply packages/components/countdown/src/countdown.vue:57]: ok I'll change it later.

**Review by kooriookami — APPROVED** [review]

File: docs/en-US/component/statistic.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ Display statistics.
...
 
-statistic/num
+statistic/basic
 
 :::
 
 ## Countdown
 
-:::demo Set `timeIndices`Start the countdown. Countdown component, support English and Chinese countdown, support to add other components control countdown.
+:::demo Countdown component, support English and Chinese countdown, support to add other components control countdown.
```
Comment (chenxch) [inline docs/en-US/component/statistic.md:20]: `support English and Chinese countdown` 
`Countdown` don't seem to care what language it is.

File: docs/en-US/component/statistic.md [hunk 3]
```diff
@@ -40,10 +40,10 @@ statistic/card
 ### Statistic Attributes
 
 | Attribute         | Description                    | Type                                                                | Default |
-|-------------------|--------------------------------|---------------------------------------------------------------------|---------|
-| value             | Numerical content              | ^[string] / ^[number]                                               | 0       |
+| ----------------- | ------------------------------ | ------------------------------------------------------------------- | ------- |
+| value             | Numerical content              | ^[number]                                                           | 0       |
 | decimal-separator | Setting the decimal point      | ^[string]                                                           | .       |
-| formatter         | Custom numerical presentation  | ^[Function]`(value: string \| number) => string \| number`          | —       |
+| formatter         | Custom numerical presentation  | ^[Function]`(value: number \| Dayjs) => string \| number`           | —       |
```
Comment (chenxch) [inline docs/en-US/component/statistic.md:46]: `value` type should be consistent with `props.value` type

File: docs/en-US/component/statistic.md (hunk 2 above)
Comment (holazz) [reply docs/en-US/component/statistic.md]: updated.

File: docs/en-US/component/statistic.md (hunk 3 above)
Comment (holazz) [reply docs/en-US/component/statistic.md]: updated.

**Review by chenxch — APPROVED** [review]

**Comment by chenxch** [comment]
@huangzhongzhong Please help with a review of the documentation UI.

**Pull request merged**